### DISCUSSION TITLE
Automated test of validity and truthness of AI generated replies

### DISCUSSION THREAD

**Discussion by lakruzz** [thetechcollective/public-discussions#20] [Open Source]
We're proposing an open source project:

It's an exploratory Open Source module extension to Python that can be used in traditional unit tests
as a natural addition to the various assertion types in `pytest`. The project explores various
techniques like heuristic, GPT’s, DNN, statistics etc to assess the validity or truth of an AI generated reply relative to the given prompt.

**Comment by MarckPetersen** [comment]
I will prepare myself by looking at more information regarding testing of AI via info: 
ISTQB Certified Tester AI Testing Explained:
https://www.youtube.com/watch?v=LFnHyjnFEHc&list=PL8Ql2_5rYPjgK0HDswJeqpATbOJ7xaGkR&index=3

Let's get working and alignment by doing a workshop ..i will book
